CMakeLists.txt

```diff
@@ -83,14 +83,15 @@ option(ENABLE_DOCS "Indicates whether documentation should be built." ON)
 option(WITH_GUI "Build GUI parts of OpenMS (TOPPView&Co). This requires QtGui." ON)
 option(NO_WEBENGINE_WIDGETS "Do not use QtWebengineWidgets. Disables Javascript views in TOPPView." OFF)
 option(WITH_HDF5 "Build HDF5 parts of OpenMS." OFF)
+option(WITH_PARQUET "Build Parquet parts of OpenMS." OFF)
+option(ENABLE_CWL "Build and validate the CWL description files for all TOPP tools." OFF)
```

Comment:
I think something went wrong in the merge against develop.
This is how it currently looks: https://github.com/OpenMS/OpenMS/blob/develop/CMakeLists.txt#L85-L91